portugues/colabora/comdiferencas/diff/pull/reenvio/watch/emprego/facebook/facebook.md

```diff
@@ -0,0 +1 @@
+O Facebook aceita o convite e também inclui e Udacity no projeto e juntos vocês disponibilizam um excelente curso sobre Git/GitHub!
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.